core/src/components/datetime/datetime.scss

```diff
@@ -243,7 +243,9 @@ ion-picker-column-internal {
 }
 
 :host .calendar-action-buttons ion-item ion-icon {
-  @include padding(0, 0, 0, 4px);
+  @include padding(4px);
```

Comment:
This change was needed otherwise the arrow would be shifted down 4px on iOS mode